aspnetcore/mvc/views/view-compilation.md

```diff
@@ -18,16 +18,18 @@ Razor files with a `.cshtml` extension are compiled at both build and publish ti
 > Runtime compilation:
 > * Isn't supported for Razor components of Blazor apps.
 > * Doesn't support [global using directives](/dotnet/csharp/whats-new/csharp-10#global-using-directives).
-> * Doesn't support [implicit using directives](/dotnet/core/tutorials/top-level-templates#implicit-using-directives)
+> * Doesn't support [implicit using directives](/dotnet/core/tutorials/top-level-templates#implicit-using-directives).
+> * Disables [.NET Hot Reload](xref:test/hot-reload).
+> * Is recommended for development, not for production.
 
 ## Razor compilation
 
 Build-time and publish-time compilation of Razor files is enabled by default by the Razor SDK. When enabled, runtime compilation complements build-time compilation, allowing Razor files to be updated if they're edited.
 
-In addition to build-time compilation, updating Razor views and Razor Pages is supported using <xref:test/hot-reload>.
+In addition to build-time compilation, updating Razor views and Razor Pages is supported using [.NET Hot Reload](xref:test/hot-reload).
 
 > [!NOTE]
-> When enabled, runtime compilation disables [.NET Hot Reload](xref:test/hot-reload).
+> When enabled, runtime compilation disables [.NET Hot Reload](xref:test/hot-reload). We recommend using Hot Reload instead of Razor runtime compilation during development, but we understand that may not be feasible because of Hot Reload's limitations. We're working to improve Hot Reload so that Razor runtime compilation will no longer be necessary, but we don't expect that to happen in .NET 8.
```

Comment:
```suggestion
> When enabled, runtime compilation disables [.NET Hot Reload](xref:test/hot-reload). We recommend using Hot Reload instead of Razor runtime compilation during development.
```